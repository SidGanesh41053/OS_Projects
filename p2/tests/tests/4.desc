Verify getparentname truncates process names when small buffers are provided
