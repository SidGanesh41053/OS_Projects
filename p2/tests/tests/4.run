cd ../solution; ../tests/run-xv6-command.exp CPUS=1 Makefile.test test_4 | grep -E 'XV6_TEST_OUTPUT|XV6_TEST_ERROR'; cd ../tests