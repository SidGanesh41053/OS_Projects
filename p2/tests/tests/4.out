XV6_TEST_OUTPUT Parent name: sh Child name: tes
